### DISCUSSION TITLE
Laragon 6 - Add php 5.6

### DISCUSSION THREAD

**Discussion by sgirard84** [leokhoa/laragon#442] [Q&A]
Hi all,

I'm using Laragon 6 with php 8.1 (php-8.1.10-Win32-vs16-x64) but I'd like to add php 5.6 for a few old websites I still need to maintain.

I first downloaded apache **httpd-2.4.38-win64-VC11**
Then php 5.6 **php-5.6.40-Win32-VC11-x64**

Do I need to install something else or edit some configs?

I'm getting this error message when I try to open any website

_Fatal error: Unknown: Failed opening required [my path] (include_path='.;C:/laragon/etc/php/pear') in Unknown on line 0_

I checked the php.ini from 8.1 and the path is the same, so this might not be a real reason of failure.

Thanks

**Comment by josealgardataborges** [comment]
C:\laragon\bin\php\php-5.6.40-Win32-VC11-x64
C:\laragon\bin\apache\httpd-2.4.38-win64-VC11

Should work just fine together :)

![image](https://user-images.githubusercontent.com/8929655/225710779-4cbb8023-a944-49cb-865d-e4bf3b3ade02.png)

make sure you created the folders and placed each uncompressed zip inside (as seen on picture above)

**Comment by sgirard84** [comment]
Humm weird then. I changed the default www location (on One Drive), I don't know if it can be the issue (but it works with 8.1). I'll try to move back to default location to see...

**Comment by sgirard84** [comment]
Update. It works when using the default www root location. I'll let the question open in case someone knows what to do, but at least I have a solution.

**Comment by josealgardataborges** [comment]
Changing PHP versions, shouldn't make problems.

Changing Apache versions can, since it deals with virtual hosts files.
You are changing the "default" www location on the respective file on "C:\Laragon\etc\apache2\sites-enabled" right?

Can you share your .conf file for that "www location"?
